index.html

```diff
@@ -759,7 +759,8 @@ <h2><dfn>SmartCardConnection</dfn> interface</h2>
 
           Promise&lt;ArrayBuffer&gt; transmit(BufferSource sendBuffer);
 
-          Promise&lt;undefined&gt; startTransaction(SmartCardTransactionCallback transaction);
+          Promise&lt;undefined&gt; startTransaction(SmartCardTransactionCallback transaction,
+              optional AbortSignal signal);
```

Comment:
Looking at your implementation CL I realized that it will be easier to add more options that affect how a transaction is created if we used an options dictionary here rather than adding an optional parameter directly.